easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -1151,27 +1151,30 @@ def sanity_check_step(self, *args, **kwargs):
 
         return (parent_success and success, parent_fail_msg + fail_msg)
 
-    def make_module_req_guess(self):
+    def make_module_step(self, *args, **kwargs):
         """
-        Define list of subdirectories to consider for updating path-like environment variables ($PATH, etc.).
+        Define list of subdirectories in search paths of module load environment
         """
-        guesses = super(PythonPackage, self).make_module_req_guess()
-
         # avoid that lib subdirs are appended to $*LIBRARY_PATH if they don't provide libraries
         # typically, only lib/pythonX.Y/site-packages should be added to $PYTHONPATH (see make_module_extra)
-        for envvar in ['LD_LIBRARY_PATH', 'LIBRARY_PATH']:
-            newlist = []
-            for subdir in guesses[envvar]:
+        for env_var in ['LD_LIBRARY_PATH', 'LIBRARY_PATH']:
+            search_paths = getattr(self.module_load_environment, env_var)
+            for lib_dir in SEARCH_PATH_LIB_DIRS:
```

Comment:
@lexming Hmm, this approach isn't identical to what we were doing in `make_module_req_guess`...

If we only consider the entries of `SEARCH_PATH_LIB_DIRS` (i.e. `lib` and `lib64`), we may be overlooking some additional directories that were added into `$LD_LIBRARY_PATH`.

Imagine an easyblock deriving from `PythonPackage` which does this in its constructor:

```
self.module_load_environment.LD_LIBRARY_PATH = ['lib/python']
```

Any reason why you've switched over to `SEARCH_PATH_LIB_DIRS` here?
I don't see a reason to...